test/utils.js

```diff
@@ -45,6 +45,9 @@ export function almostEqual(a, b, criteria) {
  * @return {number} A 64-bit signed integer.
  */
 export function getBitwise(value, dataType) {
+  if (Object.is(value, -0)) {
```

Comment:
@huningxin I caught an issue of -0, the returns of 
```js
getBitwise(-0, "float32")
```
is **2147483648n**, so the distance between **-0** and **0** is **2147483648n**, it isn't the same as the distance of **0n** between **-0** and **0**  by native C++ `compareUlp()` function.

I added this workaround to deal with **-0**. What's your opinion? Thanks

Reply:
Did you try -0.0 in native C++?

Reply:
Hi @huningxin,
I tested with -0.0 in native C++, the `GitBitwise()` function returns 2147483648n, and the distance between -0.0 and 0 is 2147483648n. 
I checked that return of `Object.is(-0.0, -0)` is `ture`, that's,  -0.0 and -0 are same value in JavaScript. 
Should we make the behave in JavaScript align with native behave? Any suggestion? Thanks.

Reply:
Thanks @huningxin. 
I will raise this special case on coming WG meeting.

Reply:
Cool. Could you please file an issue to log this case? It could be used for WG discussion. Thanks.